regression-test/conf/regression-conf.groovy

```diff
@@ -24,7 +24,6 @@ defaultDb = "regression_test"
 // init cmd like: select @@session.tx_read_only
 // at each time we connect.
 // add allowLoadLocalInfile so that the jdbc can execute mysql load data from client.
-jdbcUrl = "jdbc:mysql://127.0.0.1:9030/?useLocalSessionState=true&allowLoadLocalInfile=true"
```

Comment:
Regression test may change this line accordingly, which will skip testing tls if it is deleted.